toonz/sources/toonzqt/styleeditor.cpp

```diff
@@ -1020,79 +1258,68 @@ void ColorSlider::paintEvent(QPaintEvent *event) {
   int h = height();
 
   bool isVertical = orientation() == Qt::Vertical;
+  bool isLineHandle =
+      ColorSlider::s_slider_appearance == AbsoluteColoredLineHandle;
 
-  if (!isVertical) h -= 5;
+  if (isVertical) {
+    y += s_chandle_size / 2;
+    h -= s_chandle_size;
+    w -= 3;
+  } else {
+    x += s_chandle_size / 2;
+    w -= s_chandle_size;
+    h -= 3;
+    if (isLineHandle) {
+      y += 1;
+      h -= 2;
+    }
+  }
 
```

Comment:
Seems i should had tested both orientations, not just the default one alone.
This is one way to solve the crash:
```suggestion
  if (w < 2) w = 2;
  if (h < 2) h = 2;
  
```
it seems `makeLinearShading` doesn't like when size is 1 or lower.

Reply:
Confirmed that the suggested change resolves the crash. Thank you @justburner !